Assignments/Object oriented/Final_Project_customer.js

```diff
@@ -0,0 +1,95 @@
+CB.CloudApp.init('yegguroxoyua', '47345f24-5127-4318-abd3-b054fe11a2a2');
+
+class customer{
+	constructor(username,password){
+		this.user = username;
+		this.pass = password;
+		console.log(this.pass);
+		console.log(this.user);
+	}
+
+	signup() {
+		var user = this.user;
```

Comment:
You actually shouldn't use any `var` in classes. This would also create a global scope. Change it to a `let` if you need a temporary variable or `const` if it won't be assigned to anything else.